glide.yaml

```diff
@@ -10,3 +10,9 @@ import:
   version: ^1.1.0
   subpackages:
   - spew
+- package: github.com/Azure/azure-sdk-for-go
+  version: v19.1.0
+  subpackages:
+  - services/compute/mgmt/2018-06-01/compute
```

Comment:
@mjudeikis @kwoodson you should be adding the packages you are going to use here me thinks